webapp/init.sql

```diff
@@ -1,5 +1,5 @@
 
-INSERT INTO g_client (gc_name, gc_description, gc_client_id) VALUES ('admin app', 'Glewlwyd administration app', 'g_admin');
+INSERT INTO g_client (gc_name, gc_description, gc_client_id, gc_client_password) VALUES ('admin app', 'Glewlwyd administration app', 'g_admin', PASSWORD('password'));
```

Comment:
Agree. But the problem I encountered was when executing the SQL statement. Since the  gc_client_password field is NN field, the inset statement failed. 
I totally agree with your comments on the OAUTH. Thank you.
Best Regards
Maruthi